docs/data/data-grid/events/events.json

```diff
@@ -374,6 +367,13 @@
     "event": "MuiEvent<{}>",
     "componentProp": "onRowsScrollEnd"
   },
+  {
+    "projects": ["x-data-grid", "x-data-grid-pro", "x-data-grid-premium"],
+    "name": "scrollPositionChange",
```

Comment:
Since this event is documented in https://mui.com/x/react-data-grid/events/, we should mention this change in migration guide

Reply:
done